brave/brave-core
# Removes unneeded mobile UI code

Fixes: https://github.com/brave/brave-browser/issues/6219

Removes references that are now covered by native-ui in the android browser

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
1. Build storybook (`npm run storybook` from within `src/brave` from this PR)
2. Ensure consistent desktop UI

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
